packages/applicant/src/pages/applications/index.page.tsx

```diff
@@ -66,13 +73,32 @@ const ExistingApplications = ({ applicationData }: ApplicationsPage) => {
                         className="govuk-table__row"
                       >
                         <th scope="row" className="govuk-table__cell">
+                          <p
+                            className="govuk-!-margin-0 govuk-!-font-weight-bold"
+                            data-cy={`cy-application-link-${application.applicationName}`}
+                          >
+                            {application.applicationName}
+                          </p>
+                        </th>
+                        <td
+                          scope="row"
+                          className="govuk-table__cell"
+                          aria-describedby={`submission-link-${application.grantSubmissionId}`}
+                        >
                           {application.submissionStatus === 'SUBMITTED' ? (
-                            <p
-                              className="govuk-!-margin-0 govuk-!-font-weight-regular"
+                            <a
```

Comment:
We can deffo use nextjs `<Link>` components here and avoid reloading the whole app for these navigations.